cmd/kubelet/app/options/osflags_windows.go

```diff
@@ -0,0 +1,32 @@
+// +build windows
+
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package options
+
+import (
+	"github.com/spf13/pflag"
+)
+
+var (
+	FlRunService bool
+)
+
+// AddOSFlags adds specific OS flags
+func AddOSFlags(fs *pflag.FlagSet) {
```

Comment:
Modify this as follows when you move the destination value into `KubeletFlags`, so we're consistent with `KubeletFlags.AddFlags`:
```
func (f *KubeletFlags) addOSFlags(fs *pflag.FlagSet) {
  fs.BoolVar(&f.WindowsService, "windows-service", f.WindowsService, "Enable Windows Service Control Manager API integration")
}
```
and now call it at the beginning of `KubeletFlags.AddFlags` instead of in `cmd/kubelet/app/server.go` (I made `KubeletFlags.addOSFlags` private, since there shouldn't be a reason to call this separately from `KubeletFlags.AddFlags`).

Reply:
Done